pkg/web/apps/apps_http_handler.go

```diff
@@ -119,3 +120,48 @@ func (a *httpHandler) DisableAllAppVersionsByAppID(c echo.Context) error {
 	return c.JSON(http.StatusOK, "")
 
 }
+
+// InitClientApp stores device information and returns if the app version is disabled
+func (a *httpHandler) InitClientApp(c echo.Context) error {
+	// Transform the body request in the version struct
+	device := models.Device{}
+	err := json.NewDecoder(c.Request().Body).Decode(&device)
+
```

Comment:
See here how to parse the JSON body to the model struct.

Reply:
@camilamacedo86 I believe the current implementation is cleaner. The `Bind` function exists in the Echo framework for this reason and it is suggested on their documentation:

```go
// Handler
func(c echo.Context) (err error) {
  u := new(User)
  if err = c.Bind(u); err != nil {
    return
  }
  return c.JSON(http.StatusOK, u)
}
```

https://echo.labstack.com/guide/request

Reply:
In the endpoint `apps/:id/versions` we are parsing the JSON to object with `json.NewDecoder` and it works. If you test the Bind func from the echo framework and it works well as expected. Great !! This goes according to the customer 👍